common/rust/cctrusted_base/src/cc_type.rs

```diff
@@ -0,0 +1,80 @@
+use anyhow::*;
+use std::collections::HashMap;
+use std::path::Path;
+use std::result::Result;
+
+use crate::tee::TEE;
+use crate::tcg::TcgAlgorithmRegistry;
+use crate::tdx::tdx::Tdx;
+
+// supported TEE types
+#[derive(Clone, Eq, Hash, PartialEq)]
+pub enum TeeType {
+    PLAIN = -1,
+    TDX = 0,
+    SEV = 1,
+    CCA = 2,
+    TPM = 3,
```

Comment:
Can we move TPM to 0?

In the future, we will need to add RISCV CoVE (4).
Putting TPM between them looks weird to me.

Reply:
I think moving TPM to 0 is a good idea, since TPM is supposed to be the default option

Reply:
updated

Reply:
thanks!